web/assets/scss/main.scss

```diff
@@ -145,3 +145,7 @@ body:not(.js-enabled) .govuk-back-link {
 body.js-enabled .js-only {
     display: revert;
 }
+
+.app-08rem-font-size {
```

Comment:
Wasn't sure if it was better to be literal about this for re-use or keep it hyper-focused on where its used with something like `.app-smaller-badge-text`

Reply:
I feel like going this small is going to be frowned upon, we should try to stick to the minimum on https://design-system.service.gov.uk/styles/type-scale/